cvmfs/util/algorithm.h

```diff
@@ -9,15 +9,22 @@
 
 #include <algorithm>
 #include <cassert>
+#include <cmath>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+#include <sstream>
 #include <string>
 #include <vector>
+#define BUFFSIZE 150
```

Comment:
That can be a local constant expression in the function where it is used

Reply:
Changed.